microsoftgraph/microsoft-graph-toolkit
# Monaco Keyboard Navigation

Closes #1100 
Closes #1101 

### PR Type


Bugfix







### Description of the changes
These changes add support for navigating through **html**, **js** & **css** with tabs. Monaco supports  toggling keyboard trap off with **Ctrl-M** on Windows, which also moves focus to the next focusable element, in this case, the browser's address bar.

Moving the focus from the editor to the current active tab is a better user experience  because users can then navigate the tabs. I added an **Esc** keybinding that moves the focus from the editor to the current tab. However the default behaviour -- moving focus to the next focusable element -- kicks in when a user types **Ctrl-M** on Windows & Linux and **Ctrl-Shift-M** on MacOS

### PR checklist
- [x] Project builds (`yarn build`) and changes have been tested in at least two supported browsers (Edge + non-Chromium based browser)
- [x] All public APIs (classes, methods, etc) have been documented following the jsdoc syntax
- [x] [Stories](https://github.com/microsoftgraph/microsoft-graph-toolkit/wiki/Storybook#writing-stories) have been added and existing stories have been tested
- [x] Added appropriate [documentation](https://github.com/microsoftgraph/microsoft-graph-toolkit/wiki/Documentation). Docs PR: 
- [x] License header has been added to all new source files (`yarn setLicense`)

### Other information

Is there away of accessing the host dom from a shadow dom? @nmetulev 

## How to test this
1. Click on the **html** tab
2. Press **Esc**
3. Observe that you can tab through the tabs.
